book/website/collaboration/inclusive-events/inclusive-events-comms.md

```diff
@@ -0,0 +1,487 @@
+For example, not all households have a computer with reliable internet access which may make accessing and using online registration forms difficult. 
+
+
+(cl-inclusive-events-comms-speaker-information)=
+## Speaker Information
+If your event involves people with lived experience, you should consider how to set your speakers, presenters, and facilitators up for success.
+
+You will need to identify and invite your speakers, regardless of their lived experience, well in advance of the event and provide them with support so that they can plan and deliver a successful session.
+
+(cl-inclusive-events-comms-speaker-diversity)=
+### Inviting Speakers with Lived Experience
+Representation is incredibly important and you may want to consider how you represent your attendees in your invited speakers and event facilitators. 
+
+When inviting a speaker with lived experience to present, consider if you have the resources to appropriately support their session and preparation. 
+For example, you may need to provide details about the event ahead of when the information is scheduled to be sent to attendees. 
+This is to ensure that the speaker feels supported and is able to make an informed decision before agreeing to present at your event. 
+
+Ask the speaker if they require any support or accommodations and ensure that these are met to their satisfaction so that they are able to deliver their session effectively. 
+For example, ensuring that there is an appropriately trained and qualified sign language interpreter who is able to work with a deaf or hard of hearing speaker. 
+
+Additionally, if your event is supporting a speaker or facilitator with lived experience, consider asking how they wish to be identified in relation to their expertise and experience. 
+For example, many autistic people, and the autistic community as a collective, have expressed a preference for using "Identify-first" language which places the disability first in the description. 
+This is contrasted with "Person-first" language which emphasizes the person before the disability, for example, __Person with epilepsy_. 
+Many people feel strongly about person or identity first language, and it is important to ask that individual about their preference as part of planning their introduction. 
+
+You may notice that we have used both person-first and identity-first language has been used in this chapter. 
+How a person chooses to self-identify is entirely a personal decision, and it is the role of organizers and facilitators to support the choices of the speakers and attendees, 
+as long as it does not contravene the event's Code of Conduct. 
+Consider instead including a statement on the inclusive and compassionate use of both person-first and identity-first language when opening your event. 
+
+(cl-inclusive-events-comms-speaker-preparation)=
+### Speaker Preparation
```

Comment:
For this section I'm not 100% sure if this fits in "communication" or whether that should be part of the general conference pre-prep? At least the 1st section feels like it's more "supporting presenters"?

Reply:
I'll pull this out into a different (new) section.

Reply:
Actually I might think on this more. The heading for this section is "speaker information" with a description of 

> _If your event involves people with lived experience, you should consider how to set your speakers, presenters, and facilitators up for success.
You will need to identify and invite your speakers, regardless of their lived experience, well in advance of the event and provide them with support so that they can plan and deliver a successful session._

I'll come back to this after I've started pulling sections out.